### PR TITLE
retract v1.4.1 panic

### PR DESCRIPTION
**Pull request by trungdlp-wolffun**
Retract v1.4.1 to prevent somebody use this version and get error with `go get github.com/refraction-networking/utls@latest`

Issues: #218 , https://github.com/imroc/req/issues/263

### CODE REVIEW

**Comment by gaukas** [comment]
v1.4.1 may not be the only version that needs to be retracted. The actual breaking change has been made in v1.4.0.

File: go.mod [hunk 1]
```diff
@@ -2,6 +2,10 @@ module github.com/refraction-networking/utls
 
 go 1.20
 
+retract (
+	v1.4.1 // #218 panic on saveSessionTicket
```
Comment (gaukas) [inline go.mod:6]: Perhaps also v1.4.0?

**Review by gaukas — APPROVED** [review]
LGTM

**Comment by trungdlp-wolffun** [comment]
@gaukas Sorry, I'm late, but we can add list of versions from to like this: [v1.4.0, v1.4.2], all version between 1.4.0 to 1.4.2 will be retract.

**Comment by gaukas** [comment]
Cool, thanks for the info. I don't think v1.4.2 is problematic so let's keep it for now.

**Comment by trungdlp-wolffun** [comment]
@gaukas Just example haha 🤣

**Pull request merged**